src/senaite/jsonapi/v1/routes/registry.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from senaite.jsonapi import api
+from senaite.jsonapi import request as req
+from senaite.jsonapi.v1 import add_route
+
+
+@add_route("/senaite-config-registry", "senaite.jsonapi.v1.senaite-config-registry", methods=["GET"])
+@add_route("/registry/senaite-config-registry", "senaite.jsonapi.v1.registry.senaite-config-registry", methods=["GET"])
+def get(context, request, username=None):
```

Comment:
Please use the following registration:

```python
+@add_route("/registry", "senaite.jsonapi.v1.registry", methods=["GET"])
 +@add_route("/registry/<string:key>", "senaite.jsonapi.v1.registry", methods=["GET"])
def get(context, request, key=None):
    # return all registry items if key is None, otherwise try to fetch the registry key
    [...]
```

Reply:
@ramonski  This is a nice one. I like the idea of providing a more generic method for querying the registry by keyword instead of using hardcoded values.